Packages/MIES/MIES_Configuration.ipf

```diff
@@ -127,7 +127,7 @@ static Constant EXPCONFIG_UDATA_MAXCTRLARRAYINDEX = 100
 static Constant EXPCONFIG_JSON_INDENT = 4
 static StrConstant EXPCONFIG_FILEFILTER = "Configuration Files (*.json):.json;All Files:.*;"
 static StrConstant EXPCONFIG_CTRLGROUP_SUFFIX = " ControlGroup"
-static StrConstant EXPCONFIG_SETTINGS_FOLDER = "Settings"
+static StrConstant EXPCONFIG_SETTINGS_FOLDER = "Settings_template"
```

Comment:
This variable should stay the same as we want require users to manually fill `Settings`.

Reply:
Ok, so Settings should still be the actual configuration folder, whereas Settings_template should act as library storage (though inactive). I change that.